### PR TITLE
Move swiftsyntax/Misc gyb to syntax generator

### PR DESCRIPTION
**Pull request by kimdv**

### CODE REVIEW

**Comment by kimdv** [comment]
@swift-ci please test

**Comment by kimdv** [comment]
@swift-ci please test

File: CodeGeneration/Sources/generate-swiftsyntax/MiscFile.swift [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+          TupleExprElement(
+            expression: ArrayExprSyntax {
+              ArrayElement(
+                expression: FunctionCallExpr("\n.node(UnknownSyntax.self)"),
+                trailingComma: Token.comma)
+              ArrayElement(
+                expression: FunctionCallExpr("\n.node(TokenSyntax.self)"),
+                trailingComma: Token.comma)
+              for node in NON_BASE_SYNTAX_NODES {
+                if node.isBase {
```
Comment (kimdv) [inline CodeGeneration/Sources/generate-swiftsyntax/MiscFile.swift:45]: I'm not sure if it's a bug in the original gyb code. But this if don't make sense as the `NON_BASE_SYNTAX_NODES` don't contain any base

File: CodeGeneration/Sources/generate-swiftsyntax/MiscFile.swift [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      modifiers: [DeclModifierSyntax(name: Token.public), DeclModifier(name: Token.static)],
+      name: IdentifierPattern("structure"),
+      type: TypeAnnotation(
+        colon: .colon,
+        type: SimpleTypeIdentifier("SyntaxNodeStructure")
+      )
+    ) {
+      
+      ReturnStmt(expression: FunctionCallExpr(
+        calledExpression: MemberAccessExpr(".choices"),
```
Comment (ahoppen) [inline CodeGeneration/Sources/generate-swiftsyntax/MiscFile.swift:33]: Can you use the `callee` overload here so you don't need to specify `leftParen`?

File: CodeGeneration/Package.swift [hunk 3]
```diff
@@ -11,6 +11,7 @@ let package = Package(
     .executable(name: "generate-swiftbasicformat", targets: ["generate-swiftbasicformat"]),
     .executable(name: "generate-swiftideutils", targets: ["generate-swiftideutils"]),
     .executable(name: "generate-swiftparser", targets: ["generate-swiftparser"]),
+    .executable(name: "generate-swiftsyntax", targets: ["generate-swiftsyntax"]),
```
[the remark this answers is not in the document]
Comment (ahoppen) [reply CodeGeneration/Package.swift:14]: Thank you so much!

**Comment by kimdv** [comment]
@swift-ci please test

**Comment by kimdv** [comment]
@swift-ci please test

**Pull request merged**